packages/components/src/utils/testing/snapshot-input-testing.tsx

```diff
@@ -0,0 +1,68 @@
+import { executeSnapshotTests } from './snapshot-testing';
+
+export function executeInputSnapshotTests<Props extends Record<string, unknown>>(
+	ComponentName: string,
+	components: unknown[],
+	extendedProps?: Partial<Props>,
+	options: { hasSmartButton?: boolean } = {},
+) {
+	const baseObj: unknown = {
+		_label: 'Label',
+		_name: 'field',
+		_placeholder: 'Hier steht ein Platzhaltertext',
+
+		...(extendedProps || ({} as Props)),
```

Comment:
Wäre es hier vielleicht noch möglich, auf die Type Assertions zu verzichten, oder geht es in diesem Fall nicht ohne?

Reply:
Wie im Daily bereits kurz erwähnt, bin ich nicht ganz glücklich mit der Umsetzung, das noch die FieldInput als Type fehlt, denn im Endeffekt macht diese Methode nicht anders als die ganzen Field Spezifischen Dinge für die Input Komponente zu testen. FieldInput ist nächster Schritt